src/app/components/hris/employees/view-employee/view-employee.component.ts

```diff
@@ -425,4 +518,8 @@ export class ViewEmployeeComponent {
     const result = capitalizedWords.join(' ');
     return result;
   }
+
+  compareNames(a: string, b: string, isAsc: boolean) {
```

Comment:
Refactor: It is good practice to have descriptive variables. There are one or two above as well.

[Naming Variables Effectively for Clean Code Practices](https://www.linkedin.com/pulse/naming-variables-effectively-clean-code-practices-mominur-rahman-rynvc/)

Reply:
Resolved